spec/javascripts/evidence_confirmation_spec.coffee

```diff
@@ -0,0 +1,77 @@
+#= require radio_buttons_module
+#= require evidence_confirmation
+
+describe "EvidenceConfirmationModule", ->
+  element = null
+  beforeEach ->
+    element = $("""<div class="small-12 medium-8 large-5 columns">
```

Comment:
I'm just thinking (and I don't know the answer), if there's a way of rendering this HTML from the actual codebase, rather than having it pasted here. Because if we change the markup for that form, this test may still pass.